libs/components/tabs/src/lib/modules/tabs/tab-button.component.html

```diff
@@ -19,9 +20,10 @@
       'sky-btn-tab-disabled': disabled,
       'sky-tab-btn-closeable': closeable
     }"
-    [tabindex]="disabled ? '-1' : '0'"
+    [tabindex]="active ? '0' : '-1'"
```

Comment:
we only want to include the tab in the tabbing order if its active--the others are reached with arrow keys